frontend/test/metabase/scenarios/dashboard/dashboard.cy.spec.js

```diff
@@ -81,6 +81,30 @@ describe("scenarios > dashboard", () => {
     cy.findByText("How many orders were placed in each year?");
   });
 
+  it("should allow empty card title (metabase#12013)", () => {
+    cy.intercept("GET", "/api/table/**/query_metadata").as("tableMetadata");
+    visitDashboard(1);
+
+    // 3 tables
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
+    cy.wait("@tableMetadata");
+
+    cy.findByTextEnsureVisible("Orders");
+    cy.get("[data-testid='legend-caption']").should("exist");
+
+    cy.findByTextEnsureVisible("Quantity").realHover();
```

Comment:
What is this `realHover` doing? I don't see it triggering anything and also I don't see any assertion after that action o.O

Reply:
Oh... you're triggering the hover over the dashboard card here? But why before you even click on the edit icon?
It's just a coincidence that it works. Please don't do that.

Reply:
It's to show the dashboard action. But I just realize I should use the helper `showDashboardCardActions` instead.